apps/api_web/mix.exs

```diff
@@ -72,8 +72,8 @@ defmodule ApiWeb.Mixfile do
       {:benchwarmer, "~> 0.0", only: [:dev, :test]},
       {:dialyxir, ">= 0.0.0", only: [:dev, :test]},
       {:logster, "~> 1.0"},
-      {:phoenix_swagger, "~> 0.8.1"},
-      {:ex_json_schema, "< 0.6.0"},
+      {:phoenix_swagger, github: "mbta/phoenix_swagger", branch: "lev-elixir19"},
```

Comment:
Should change branch to specific commit ref once https://github.com/mbta/phoenix_swagger/pull/2 lands